ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh

```diff
@@ -50,5 +50,6 @@ if [ -f "${OPERATOR_DIR}/tests/scorecard/config.yaml" ]; then
     --namespace="${NAMESPACE}" \
     --kubeconfig "${KUBECONFIG}" \
     --output json \
+    --wait-time 3000s \
```

Comment:
Please add a comment why such value was used and ideally explanation why it should not be increased

Reply:
Added the explanation above the command

Reply:
LGTM